Button/Button.js

```diff
@@ -146,6 +165,15 @@ const ButtonBase = kind({
 		 */
 		joinedPosition: PropTypes.oneOf(['left', 'center', 'right']),
 
+		/**
+		 * Boolean controlling whether this component should enforce the "minimum width" rules.
+		 *
+		 * @type {Boolean}
+		 * @default true
```

Comment:
I don't know why @default is true here? It might make a developer confused. Let's remove it.
NOTE: Need to fix in sandstone